src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -667,13 +667,31 @@
         [:null name]
         [field name]))))
 
+(defn merge-function-options
+  "Optionally merge call and grouping into an existing query map.
+   Alias function calls with escape-quoted function name for hsqldb compatability."
+  [m call grouping]
+
+  (if-not call
+    m
+    (let [fn-call (apply hcore/call call)
+          fn-alias (format "\"%s\"" (name (first call)))]
```

Comment:
I don't think this is necessary, you can just put `(first call)` and the `fn-alias` and honeysql will know what to do.